src/x509.cc

```diff
@@ -45,6 +45,85 @@ std::string parse_args(const Nan::FunctionCallbackInfo<v8::Value>& info) {
   return *String::Utf8Value(info[0]->ToString());
 }
 
+
+
+NAN_METHOD(verify) {
+  Nan::HandleScope scope;
+  OpenSSL_add_all_algorithms();
```

Comment:
I would say to leave that line out entirely and note in the documentation that it may only work on OpenSSL 1.1.0 or greater. Most people should have already upgraded to 1.1.0, and those who haven't should, due to the heartbleed vulnerability that was discovered.